sdk/c/loader.c

```diff
@@ -24,6 +24,8 @@
 #include <openssl/engine.h>
 #include <openssl/conf.h>
 
+#define strncpy_s strncpy
```

Comment:
If this passes a scanning tool... we need a better scanning tool. There are still occurrences of `strncpy` in this file, those should be updated.